doc/lpc18/background.tex

```diff
@@ -176,7 +176,21 @@ \subsubsection{Concurrency}
 
 \subsection{XDP: eXpress data path}
 
-TODO.
+The core idea behind XDP is optimizing cases where relatively simple
```

Comment:
I think that you are repeating this phrase in the next paragraph to a large degree. Perhaps you should only keep that one.

Reply:
ok! thanks